.evergreen/atlas_data_lake/drivers-test-local.yaml

```diff
@@ -0,0 +1,66 @@
+environment: development
+
+hosting:
+  frontend:
+    bindAddrs: ["localhost:27017"]
+    cursor:
+      metadata:
+        memory: true
+  showInternalErrors: true
+
+execution:
+  mqlrunPath: ${MONGOHOUSE_MQLRUN}
+
+tenants:
+  config:
+    inline:
+      hostnames:
+        - localhost
+      security:
+        roles:
+          # Based on the atlasAdmin custom role defined in NDSDBUser.java in the
+          # 10gen/mms repo.
+          - role: atlasAdmin
+            roles:
+              - role: readWriteAnyDatabase
+              - role: enableSharding
+              - role: dbAdminAnyDatabase
+              - role: backup
+              - role: clusterMonitor
+            privileges:
+              - actions: ["viewRole", "moveChunk", "splitChunk", "viewUser"]
+              - actions: ["flushRouterConfig", "storageGetConfig", "storageSetConfig"]
+                resource:
+                  cluster: true
+        users:
+          - name: mhuser # password: "pencil"
+            credentials:
+              scramSHA1:
+                iterations: 10000
+                salt: "CVmjg1IDp3KfHfiXNZhmbQ=="
+                storedKey: "9cIonYrtLI/+OmZ96wiQkduwXeI="
+                serverKey: "kCfqkc7sgQaihzXHqbxZxz3O3H4="
+              scramSHA256:
+                iterations: 15000
+                salt: "6i+wBCHE21EaI8IF9+tvPkCoYWY+IyedLaVXGQ=="
+                storedKey: "hnn0iJZ6ZNO9X34oyxtZUKLAMScwpCdch3vGPuDXOvM="
+                serverKey: "3Qg6ptTiTvPwhGSZi0NpAkmsW8P5C/J46Is1t7xaIho="
+            roles:
+              - role: atlasAdmin
+      storage:
+        stores:
+          - name: localstore
+            provider: local
+            path: "${DRIVERS_TOOLS}/.evergreen/atlas_data_lake/testdata/stores/drivers-tests"
```

Comment:
If I understand correctly, `stores` need only refer to a single directory. Can we just use `.evergreen/atlas_data_lake/stores` instead and then use `test.driverdata.json` as the path below? That might let us eliminate the `/test/*` wildcard and cryptic `1.json` filename.

I understand why mongohouse supports wildcards to populate a collection with data from many sources, but I don't think we actually need to utilize that in our testing (and aren't, if we're just using a single JSON file anyway), so I'd prefer to have a simpler directory structure in the interest of future maintainers.